akka-stream/src/main/scala/akka/stream/OverflowStrategy.scala

```diff
@@ -55,6 +57,11 @@ object OverflowStrategy {
   def dropBuffer: OverflowStrategy = DropBuffer
 
   /**
+   * If the buffer is full when a new element arrives, drops the new element.
+   */
+  def dropNew: OverflowStrategy = DropNew
```

Comment:
![image](https://cloud.githubusercontent.com/assets/422086/7928322/bee13cd8-08f6-11e5-8fde-d1a2990966da.png)

Both sound alright for me. `dropNew` is a tad shorted though.